ch3/ex06.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python
+# EX 06: create a dictionary with 3 random keys and 3 random strings as elements
+
+import random
+import string
+
+CHOICE = string.ascii_letters + string.digits
+
+def randomstring(times, chars):
```

Comment:
well, is it to early to speak about import and DRY? You wrote this function 4 times in 4 different files... You should learn how to have it in a separate file and import it